beacon-chain/core/blocks/deposit.go

```diff
@@ -55,7 +64,7 @@ func ProcessPreGenesisDeposits(
 			return nil, err
 		}
 	}
-	return beaconState, nil
+	return nil, nil
```

Comment:
This function should not return a beacon state at all.

Reply:
I think you meant it should

Reply:
Nvm 🤦